frontend/src/Admin/components/NavBar.js

```diff
@@ -51,3 +57,13 @@ export default class NavBar extends Component {
     )
   }
 }
+
+
+const mapDispatchToProps = (dispatch) => ({
+  switchToJudge: () => dispatch(switchToJudge())
```

Comment:
Can you stick this instead in [`frontend/src/Admin/containers/NavBar.js`](https://github.com/robmcl4/A-B-S-T-R-A-C-T-I-O-N/blob/master/frontend/src/Admin/containers/NavBar.js#L8), we've been leaving the "components" folder free of redux hookup